doc/user/depends.xml

```diff
@@ -109,26 +109,26 @@ int main() { printf("Hello, world!\n"); }
     things when an input file changes,
     so that the built software is up to date.
     By default,
-    &SCons; keeps track of this through an
-    MD5 &signature;, or checksum, of the contents of each file,
+    &SCons; keeps track of this through a
+    <firstterm>content signature</firstterm>,
```

Comment:
can we change this to use an entity?  I see was using &signature before?

Reply:
maybe if *content signature* was the entity?

So, you made me look... right now we have two that don't exactly fit - and `<literal>` is not really the way those two should be anyway, they're not programmatic elements like variables, functions, command lines, etc.
```
<!ENTITY signature "<literal xmlns='http://www.scons.org/dbxsd/v1.0'>signature</literal>">
<!ENTITY buildsignature "<literal xmlns='http://www.scons.org/dbxsd/v1.0'>build signature</literal>">
```

Reply:
sounds reasonable? content signature as a new entity?